<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(207, 207, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="file_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="MplWidget" name="matrix_fig" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>600</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="file_lbl">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>561</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>File PEPICO matrix</string>
    </property>
   </widget>
   <widget class="MplWidget" name="msPES_fig" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>600</width>
      <height>230</height>
     </rect>
    </property>
   </widget>
   <widget class="MplWidget" name="TOF_fig" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>230</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m1_ent">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>80</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m2_ent">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>105</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>56</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t1_ent">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>130</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2.320</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t2_ent">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>155</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4.308</string>
    </property>
   </widget>
   <widget class="QLabel" name="m1_lbl">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>85</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>m1 (amu):</string>
    </property>
   </widget>
   <widget class="QLabel" name="m2_lbl">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>m2 (amu):</string>
    </property>
   </widget>
   <widget class="QLabel" name="m3_lbl">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>135</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>tof1 (us):</string>
    </property>
   </widget>
   <widget class="QLabel" name="m4_lbl">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>tof2 (us):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tof_ms_btn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>190</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot MS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tof_lim2_ent">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>670</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tof_lim1_ent">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>670</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="int_lbl">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>670</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>integration range (us):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="en_lim1_ent">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="en_lim2_ent">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>310</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>92</string>
    </property>
   </widget>
   <widget class="QLabel" name="int_lbl_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>integration range (eV):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_msPES_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>670</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save ms-PES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_esTOF_btn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>350</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save es-TOF and es-MS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
